lib/AstOfLlbc.ml

```diff
@@ -868,16 +1181,30 @@ let rec expression_of_raw_statement (env: env) (ret_var: C.var_id) (s: C.raw_sta
       Krml.Helpers.with_unit K.(EAssign (dest, maybe_addrof env ty (with_type t (EBufRead (e1, e2)))))
 
   | Call { func = FnOpRegular fn_ptr; args; dest; _ }
-    when false && Charon.NameMatcher.match_fn_ptr env.name_ctx RustNames.config RustNames.from fn_ptr ->
+    when (
+      Charon.NameMatcher.match_fn_ptr env.name_ctx RustNames.config RustNames.from_u16 fn_ptr ||
```

Comment:
You can put those in a map:
https://github.com/AeneasVerif/charon/blob/ed5047fd07e6c6aa8e7b9f352637f91977adacbe/charon-ml/src/NameMatcher.ml#L1186

Reply:
Created https://github.com/AeneasVerif/eurydice/issues/20 as a followup